python/lsst/obs/base/ingest.py

```diff
@@ -848,7 +855,22 @@ def makeDependencyRecords(
         records : `dict` [`str`, `DimensionRecord`]
             The records to insert, indexed by dimension name.
         """
-        return {}
+        records = {}
+        exposure = universe["exposure"]
+        if "group" in exposure.implied:
+            records["group"] = universe["group"].RecordClass(
+                name=obsInfo.exposure_group,
+                instrument=obsInfo.instrument,
+            )
+        if "day_obs" in exposure.implied:
+            instrument_record = self._instrument_records[obsInfo.instrument]
+            offset = instrument_record.day_obs_offset
```

Comment:
I do think writing `None` for lab data is a bit better that writing a timespan that won't have anything like the usual relationship with the timespans of the related exposures, because it makes it explicit that the value should not be used, but:

 - I can't think of any concrete case where it'd cause a problem;
 - we can't assume any kind of strict sub-internal relationship between `day_obs.timespan` and `exposure.timespan` anyway, since exposures can cross the `day_obs` boundary.

So I'm happy to leave it up to you what we do here.

(I'm just starting the rest of the review now; thought I'd weigh in on this first.